### ISSUE TITLE
Feature to accept categorical input

### ISSUE THREAD

**Issue by sophianguyen189** [scikit-learn-contrib/hiclass#82]
Hi @mirand863,

I wonder if you have any plan to add feature to accept categorical inputs which allows users to leverage optimal split for categorical data in base classifier like catboost, lightgbm?

Thanks

**Comment by mirand863** [comment]
Hi @sophianguyen189,

Sorry for the late reply. I tried to look a bit into it, but I am not sure I understand the application of categorical inputs. What would be the advantage?

Kind regards,
Fabio
